.github/workflows/vast.yaml

```diff
@@ -227,6 +232,22 @@ jobs:
         run: |
           cmake --build "$BUILD_DIR" --target package
 
+      - name: Verify Docker Image
+        if: matrix.os.tag == 'Ubuntu' && matrix.os.compiler == 'GCC' && matrix.configure.tag == 'Release'
```

Comment:
We have this exact check in a few places now. Can you add a `matrix.use_for_docker` key that is true for this case and false for all others?